API/ev3_sensor.c

```diff
@@ -372,6 +395,17 @@ int ReadSensor(int sensorPort)
 				return (-1)*(((help>>4) & 0xFF)*10 + ((help & 0xF) * 10 / 15)) * 9/5 + 320;
 			}
 			return (((help>>4) & 0xFF)*10 + ((help & 0xF) * 10 / 15)) * 9/5 + 320;
+		//New for HiTechnic IRSEEKER V2
+		case HT_DIR_DC:
+			return *((DATA16*)data);
+		case HT_DIR_AC:
+			return *((DATA16*)data);
+		case HT_DIR_DALL:
+			return *((DATA16*)data);
+		case HT_DIR_AALL:
+			return *((DATA16*)data);
+		case HT_COMP:
+			return *((DATA16*)data)*2;
```

Comment:
What's the `*2` for?

Reply:
I'll ask why this is here. Might be the sensor using 2 degree steps or something like that.

Reply:
"The sensor outputs a relative value between 0 and 180. *2 is supposed to give an absolute value" 
I am not sure about how to deal with this. If the sensor outputs 0-180, it might be misleading return something else here. Also I did not look further into the relative/absolute part and what exactly the sensor outputs.

From what I've read in gitter, there might be another PR on the way that implements the same sensor. If that is the case, it might be best to just scrap the HT_COMP sensor from this one since it seems to be a quite minimal implementation.

I'll leave this open and wait until the sensor is confirmed or denied. Then we can think about how to continue.